### ISSUE TITLE
Make the beeping sound shorter

### ISSUE THREAD

**Issue by log101** [naaando/rush#32]
Hi, I really enjoy using the app but the sound when you're done studying is really too long and annoying. It beeps 5 times, so if you make it 3, I think it would suffice.

I'm on elementary OS 5.1.7 Hera using the latest version of Rush from the app center.

**Comment by quentincaffeino** [comment]
Agree, tried doing it myself, but can't wrap my head around how libcanberra works (lib used to play system sounds). Maybe it had to be done by replacing with custom sound.

**Comment by sm-wilson** [comment]
I'd also recommend an option of different sounds that aren't as jarring. The beep is MUCH louder than other system sounds, and I nearly jumped out of my chair the first few times it went off.
